pkg/controllers/repositoryserver/handler.go

```diff
@@ -312,3 +300,23 @@ func (h *RepoServerHandler) waitForPodReady(ctx context.Context, pod *corev1.Pod
 	}
 	return nil
 }
+
+func (h *RepoServerHandler) updateProgressInCRStatus(ctx context.Context, progress crv1alpha1.RepositoryServerProgress) error {
+	h.Logger.Info("Fetch latest version of RepositoryServer to update the ServerInfo in its status")
```

Comment:
1. Adding these logs at the Info level creates too much noise when there are n number of CRs
2. These logs are redundant. It doesn't provide much info while debugging
3. Cannot differentiate logs for multiple CRs from this format, which make it of no use while debugging.

This comment is applicable for other occurrences of logs as well.

Reply:
1. I agree that this will create noise when creating multiple CRs and we should reduce number of logs. We atleast should add the logs when the repository server CR is created/deleted/updated, since I see similar logs when we create blueprints/actionsets.
 
2. These logs are not useless and we can easily differentiate between different CRs as the `logger` object we get is from the context of reconcile function, hence it prints which CR this log is for:
eg: 
```
{"level":"debug","ts":"2023-07-04T15:45:05Z","msg":"Update pod name in RepositoryServer /status","controller":"repositoryserver","controllerGroup":"cr.kanister.io","controllerKind":"RepositoryServer","RepositoryServer":{"name":"kopia-repo-server","namespace":"kanister"},"namespace":"kanister","name":"kopia-repo-server","reconcileID":"b5e5030d-0665-4c71-a8eb-78de738c8f0d"}
{"level":"debug","ts":"2023-07-04T15:45:05Z","msg":"Fetch latest version of RepositoryServer to update the ServerInfo in its status","controller":"repositoryserver","controllerGroup":"cr.kanister.io","controllerKind":"RepositoryServer","RepositoryServer":{"name":"kopia-repo-server","namespace":"kanister"},"namespace":"kanister","name":"kopia-repo-server","reconcileID":"b5e5030d-0665-4c71-a8eb-78de738c8f0d"}

```

Reply:
I am not against adding logs for logging events. But it should have more context with it so that they can be differentiated during debugging. If the logger adds the required fields like CR name/namespaces then that works. But at the same time, we should avoid adding too many logs, that would just create chaos

Reply:
I agree